maps/DeadSpace/M.03 Ishimura (L).dmm

```diff
@@ -26800,13 +26800,13 @@
 /turf/simulated/floor/dank/golf_brown,
 /area/ishimura/lower/mining/processing_room)
 "bdy" = (
-/obj/machinery/input,
 /obj/effect/floor_decal/industrial/loading{
 	dir = 4
 	},
 /obj/machinery/light{
 	dir = 1
 	},
+/obj/machinery/input/ore,
```

Comment:
why does a new machine need to be put in?

Not that i'd mind, but mapping changes can cause conflicts with other ongoing changes to the same map

Reply:
`/ore` subtype calls `wake_up()` in `Uncrossed()` too